qutebrowser/config/configdata.yml

```diff
@@ -33,7 +33,7 @@ history_gap_interval:
     Items with less time between them are grouped when being displayed in `:history`.
     Use -1 to disable separation.
 
-ignore_case:
+search.ignore_case:
```

Comment:
I think you want to add a `renamed:` argument so the autoconfig can be migrated properly for this key